api/SpeechSynthesisEvent.json

```
@@ -238,10 +238,63 @@
             }
           },
           "status": {
-            "experimental": true,
+            "experimental": false,
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "milliseconds": {
+          "__compat": {
+            "description": "<code>elapsedTime</code> in milliseconds",
+            "support": {
+              "chrome": {
+                "version_added": "33"
+              },
+              "chrome_android": {
+                "version_added": "33"
+              },
+              "edge": {
+                "version_added": "14"
+              },
+              "firefox": {
+                "version_added": "49",
+                "version_removed": "95",
+                "partial_implementation": true,
+                "notes": "Windows and macOS only."
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": "21"
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": "7",
+                "version_removed": "14.1"
+              },
+              "safari_ios": {
+                "version_added": "7",
+                "version_removed": "14.5"
+              },
+              "samsunginternet_android": {
+                "version_added": "3.0"
+              },
+              "webview_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
```

This has been removed from the spec, so…

```suggestion
              "standard_track": false,
```